cargo-typify/src/lib.rs

```diff
@@ -38,6 +42,7 @@ pub struct CliArgs {
 }
 
 impl CliArgs {
+    /// Output path.
     pub fn output_path(&self) -> Option<PathBuf> {
```

Comment:
probably just nothing in this file should be `pub` since it's a command...

Reply:
`CliArgs`'s struct fields should be documented as those docs appear on the command line `--help`.
IMO it is simpler to write some basic docs, rather than having guidelines for contributors when `#[allow(missing_docs)]` is allowed and when it isn't.

Reply:
Good point!